cmd/virt-handler/virt-handler.go

```diff
@@ -397,7 +440,16 @@ func (app *virtHandlerApp) Run() {
 		panic(fmt.Errorf("failed to detect the presence of selinux: %v", err))
 	}
 
-	cache.WaitForCacheSync(stop, vmiSourceInformer.HasSynced, factory.CRD().HasSynced, factory.KubeVirt().HasSynced)
+	cache.WaitForCacheSync(
+		stop,
+		vmiInformer.HasSynced,
+		vmiSourceInformer.HasSynced,
+		vmiTargetInformer.HasSynced,
+		vmiInformer.HasSynced,
+		domainSharedInformer.HasSynced,
+		factory.CRD().HasSynced,
+		factory.KubeVirt().HasSynced,
+	)
```

Comment:
nit: twice `vmiInformer.HasSynced`

In general, I think the only informers that need to be blockers here are the following:
- `vmiInformer` (used in the new vmi controller, and the waitForCacheSync should be made there)
- `vmiSourceInformer` (used in lifecycle handler, metrics, and dw-metrics)
- `factory.CRD()`
- `factory.KubeVirt()`

IOW: 
`vmiTargetInformer` is used in the migration-target controller, and the wait for sync is performed there.
`domainSharedInformer` is the same as the previous one.